src/components/InvalidVinMessage/InvalidVinMessage.js

```diff
@@ -0,0 +1,17 @@
+import StyledInvalidVinMessage from '../InvalidVinMessage/StyledInvalidVinMessage';
+import Typography from '../../../styles/Typography';
+import useStore from '../../lib/hooks/useStore';
+
+export default function DisplayInvalidVinMessage() {
```

Comment:
The component function name should match the filename.

```
export default function InvalidVinMessage(){
}
```

Reply:
adjusted accodingly.